great_expectations/self_check/util.py

```diff
@@ -805,6 +806,7 @@ def _get_test_validator_with_data_spark(  # noqa: C901 - 19
         "BooleanType": pyspark.types.BooleanType,
         "DataType": pyspark.types.DataType,
         "NullType": pyspark.types.NullType,
+        "DecimalType": partial(pyspark.types.DecimalType, 38, 18),
```

Comment:
@TrangPham The use of `partial` is new for us -- so a comment or a docstring explaining what is being done here would be very helpful.  Also, are we sure about the precision having to be `(38, 18)`?  A comment or a docstring would be nice.  Thank you.